cabal.project

```diff
@@ -160,17 +160,11 @@ allow-newer:
   monoidal-containers:aeson,
```

Comment:
I think this could also be removed, see https://hackage.haskell.org/package/monoidal-containers-0.6.2.0/dependencies